Example/Example/Purchase/CartViewController.swift

```diff
@@ -20,13 +19,29 @@ final class CartViewController: UIViewController, UITableViewDataSource {
   private let titleSubtitleCellIdentifier = String(describing: TitleSubtitleCell.self)
   private let eventHandler: (Event) -> Void
 
+  private var checkoutOption: CheckoutPickerOption = .v1Checkout {
+    didSet {
+      updateViewState()
+    }
+  }
+
   private lazy var cashButton = CashAppPayButton(size: .large) { [weak self] in
```

Comment:
It's not clear to me how to enable this button? @ScottAntonacAP

Reply:
There is room for improvement in the readability here.

Currently when the CartView loads, this line executes: `eventHandler(.cartDidLoad(self.cashButton))`. Follow this to the `PurchaseFlowController` which then runs `logicController.retrieveCashAppToken(cashButton: button)`. This runs `setCashButtonEnabled(false)`.

When `CashAppPayState` is in a ready to auth state, it then runs `setCashButtonEnabled(true)`